cli/client_test.go

```diff
@@ -237,6 +237,25 @@ func TestGetSupportEmailAction(t *testing.T) {
 	test.That(t, out.messages[0], test.ShouldContainSubstring, "test-email")
 }
 
+func TestBillingServiceDisableAction(t *testing.T) {
```

Comment:
[super nit] could add error case for empty org since you return an error for the empty org

Reply:
We actually don't test that here in this internal function because this is called by the caller of this function -- its a little hard to test but because of the way that we mock flags but ill manually make sure that its fine

Reply:
Sounds good